client/hosting/server-settings/components/sftp-card/index.tsx

```diff
@@ -95,12 +94,6 @@ export const SftpCard = ( {
 
 	const sshConnectString = `ssh ${ username }@sftp.wp.com`;
 
-	const onDestroy = () => {
-		if ( password ) {
-			removePasswordFromState( siteId, currentUserId, username );
```

Comment:
It's the case (4-th point from "Proposed Changes"), where we had a bug and TSX helped to highlight it - the function expects `siteId, username`, but we provide `siteId, currentUserId, username`.